force-app/main/default/classes/UTIL_OrgTelemetry.cls

```diff
@@ -424,4 +454,68 @@ public without sharing class UTIL_OrgTelemetry {
             objectMetrics.numAutoEnrollmentAffMappings
         );
     }
-}
+
+    /**
+     * @description Returns true if household accounts without contacts are marked
+     * for deletion.
+     */
+    @TestVisible
+    private Boolean hasAutoDeletionHouseholdAccts(Hierarchy_Settings__c edaSettings) {
+        return this.hasContactAutoDeletionEnabled(edaSettings, Util_Describe.getHhAccRecTypeID());
+    }
+
+    /**
+     * @description Returns true if academic program accounts without contacts are marked
+     * for deletion.
+     */
+    @TestVisible
+    private Boolean hasAutoDeletionProgramAccts(Hierarchy_Settings__c edaSettings) {
+        return this.hasContactAutoDeletionEnabled(edaSettings, Util_Describe.getAcademicAccRecTypeID());
+    }
+
+    /**
+     * @description Returns true if educational institution accounts without contacts are marked
+     * for deletion.
+     */
+    @TestVisible
+    private Boolean hasAutoDeletionEducationalInstitutionAccts(Hierarchy_Settings__c edaSettings) {
+        return this.hasContactAutoDeletionEnabled(edaSettings, Util_Describe.getEducationalInstitutionAccRecTypeID());
+    }
+
+    /**
+     * @description Returns true if business organization accounts without contacts are marked
+     * for deletion.
+     */
+    @TestVisible
+    private Boolean hasAutoDeletionBusinessOrgAccts(Hierarchy_Settings__c edaSettings) {
+        return this.hasContactAutoDeletionEnabled(edaSettings, Util_Describe.getBizAccRecTypeID());
+    }
+
+    /**
+     * @description Returns true if university department accounts without contacts are marked
+     * for deletion.
+     */
+    @TestVisible
+    private Boolean hasAutoDeletionUniversityDeptAccts(Hierarchy_Settings__c edaSettings) {
+        return this.hasContactAutoDeletionEnabled(edaSettings, Util_Describe.getUniversityAccRecTypeID());
+    }
+
+    /**
+     * @description Returns true if the specified account record type is enabled for contacts
+     * auto-deletion.
+     */
+    private Boolean hasContactAutoDeletionEnabled(Hierarchy_Settings__c edaSettings, String accountRT) {
```

Comment:
Suggestion (optional) to use `is` for Boolean values and to shorten the method:
```
    private Boolean hasContactAutoDeletionEnabled(Hierarchy_Settings__c edaSettings, String accountRT) {
        
        Boolean isAutoDeleteConfigured = edaSettings.Accounts_to_Delete__c != null;
        Boolean isRecordTypeDefined = accountRT != null;

        if (isAutoDeleteConfigured && isRecordTypeDefined) {
            return edaSettings.Accounts_to_Delete__c.contains(accountRT);

        } else {
            return false;
        }
    }
```

Reply:
Sure thing ([reference commit](https://github.com/SalesforceFoundation/EDA/pull/1934/commits/4878a17f73d9dccbf878752452927a9f1aa92020)).